app/config/portal/PortalUrlConfig.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config.portal
+
+import javax.inject.Inject
+import play.api.i18n.MessagesApi
+import play.api.mvc.Request
+
+class PortalUrlConfig @Inject()(implicit private val messagesApi: MessagesApi) {
+
+  val vat: Vat = new Vat
```

Comment:
Feels a bit unnecessary since there would be nothing here other than VAT on this service

but only refactor this if time permits

Reply:
:+1: